Source/FolioReaderContainer.swift

```diff
@@ -9,59 +9,76 @@
 import UIKit
 import FontBlaster
 
-var readerConfig: FolioReaderConfig!
-var book: FRBook!
-
 /// Reader container
-open class FolioReaderContainer: UIViewController {
-    var centerNavigationController: UINavigationController!
-	var centerViewController: FolioReaderCenter!
-    var audioPlayer: FolioReaderAudioPlayer!
-    var shouldHideStatusBar = true
-    var shouldRemoveEpub = true
-    var epubPath: String!
-    fileprivate var errorOnLoad = false
+open class FolioReaderContainer     : UIViewController {
+
+    var centerNavigationController  : UINavigationController?
+    var centerViewController        : FolioReaderCenter?
+    var audioPlayer                 : FolioReaderAudioPlayer?
+    var shouldHideStatusBar         = true
+    var shouldRemoveEpub            = true
+    var epubPath                    : String
+    var book                        : FRBook
```

Comment:
Even though that is not our way at all, I removed every single unnecessary white spaces on the variable declarations in the code. The current comments will be "done" once we merge the related branch into the `master` one.